website/static/js/profile.js

```diff
@@ -260,14 +277,21 @@ BaseViewModel.prototype.handleSuccess = function() {
     }
 };
 
-BaseViewModel.prototype.handleError = function(response) {
+BaseViewModel.prototype.handleError = function(response) {    
     var defaultMsg = 'Could not update settings';
-    var msg = response.responseJSON.message_long || defaultMsg;
-    this.changeMessage(
-        msg,
-        'text-danger',
+    var msg;
+    if (typeof response.responseJSON != "undefined") {
```

Comment:
OK, accessing `responseJSON` is correct.

Consult https://github.com/rwaldron/idiomatic.js#type for info on proper type checking.

```js
response.responseJSON === undefined
```